.cirrus.yml

```diff
@@ -0,0 +1,21 @@
+package_ci_task:
+  timeout_in: 120m
+  container:
+    dockerfile: ci/Dockerfile
+    matrix:
+      - docker_arguments:
+        - ZEEK_LTS: 1
+        - ZEEK_VERSION: 3.0.13-0
+      - docker_arguments:
+        - ZEEK_LTS:
+        - ZEEK_VERSION: 3.2.4-0
+      # TODO(bbannier): add zeek-4.0 LTS release.
```

Comment:
The packages of the non-LTS version will switch to 4.0.0 later today; LTS will stay at 3.0.x for another 2 weeks or so.

Let's update the non-LTS to point to the latest later today.